### PR TITLE
fixed the redirect on GitHub login from welcome page

### LINKED ISSUE #243: Integrated new Signup Flow
## Issue
When signing in with GitHub send the page href in query params as `&state=`
- RFC - https://github.com/Real-Dev-Squad/website-backend/issues/688

### Description
When you visit the `main site` and click on login, you are redirected to the original GitHub login page. We want to emulate a similar scenario for the `welcome site`.

### Expected Flow
- User visits `welcome-site` for the first time
- User clicks on `sign-in-with-github`
- Now, they are redirected to the GitHub login page
- once the login is successful the user is expected to land back to the `welcome site`
- Example PR - https://github.com/Real-Dev-Squad/website-www/pull/277

### PR DESCRIPTION
**Pull request by AkashsRepositories**
Added the &state parameter and gave a value of welcome page URL in GitHub OAuth URL.
Now if user login from the Welcome page of the realdevsquad website after successful login, they will be redirected to the Welcome page.

Closes #243

### CODE REVIEW

File: js/navbar.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ window.addEventListener('DOMContentLoaded', (event) => {
     checkURL === productionURL
       ? '23c78f66ab7964e5ef97'
       : 'c4a84431feaf604e89d1';
-  userLoginEl.href = `https://github.com/login/oauth/authorize?client_id=${clientId}`;
+
+  userLoginEl.href = `https://github.com/login/oauth/authorize?client_id=${clientId}&state=${productionURL}`;
```
Comment (ivinayakg) [inline js/navbar.js:10]: Instead of productionURL use checkURL. 
To make sure even the pages are covered with this feature.

**Review by ivinayakg — CHANGES_REQUESTED** [review]

File: js/navbar.js (hunk 1 above)
Comment (AkashsRepositories) [reply js/navbar.js:10]: Sure.

Comment (AkashsRepositories) [reply js/navbar.js]: Made suggested changes. Please check.

**Review by ivinayakg — APPROVED** [review]
LGTM
Good work 👍
Thank you for making the suggesting changes.

**Comment by AkashsRepositories** [comment]
> LGTM Good work 👍 Thank you for making the suggesting changes.

Thanks for assigning this issue to me. Looking forward for more contributions and learnings with them 😇

**Comment by AkashsRepositories** [comment]
Can you please merge this PR? @ankushdharkar

**Pull request merged**